### PR TITLE
Added gmaps for economic development connector.

### PR DESCRIPTION
**Pull request by gunjanbaid**

### CODE REVIEW

File: user-image/Dockerfile [hunk 1]
```diff
@@ -68,6 +68,10 @@ RUN pip install git+https://github.com/data-8/nbresuse.git@2f9144f && \
 RUN pip install git+https://github.com/data-8/gitautosync@b32a087 && \
 	jupyter serverextension enable  --sys-prefix --py nbgitautosync
 
+# gmaps notebook extension
+RUN pip install gmaps && \
```
Comment (yuvipanda) [inline user-image/Dockerfile]: should be 'pip install --no-cache-dir'

Comment (yuvipanda) [inline user-image/Dockerfile]: Also add a comment on which connector needs this?

File: user-image/Dockerfile [hunk 2]
```diff
@@ -68,6 +68,11 @@ RUN pip install git+https://github.com/data-8/nbresuse.git@2f9144f && \
 RUN pip install git+https://github.com/data-8/gitautosync@b32a087 && \
 	jupyter serverextension enable  --sys-prefix --py nbgitautosync
 
+# gmaps notebook extension 
+# for ls 88-4, economic development connector
+RUN pip install gmaps && \
```
Comment (yuvipanda) [inline user-image/Dockerfile]: you missed the --no-cache-dir for pip :)

**Comment by gunjanbaid** [comment]
@yuvipanda ok to merge to staging?

**Comment by yuvipanda** [comment]
Yes feel free to do the whole process.

**Pull request merged**